packages/core/src/visit-meta-css-dependencies.ts

```diff
@@ -5,7 +5,8 @@ import type { StylableResolver } from './stylable-resolver';
 export function visitMetaCSSDependenciesBFS(
     meta: StylableMeta,
     onMetaDependency: (meta: StylableMeta, imported: Imported, depth: number) => void,
-    resolver: StylableResolver
+    resolver: StylableResolver,
+    onJsDependency?: (resolvedPath: string, imported: Imported) => void
```

Comment:
why split the "onDependency" into 2 callbacks? wouldn't all usages benefit from also registering js dependencies (with a small change to the signature)?

Reply:
This method is used externally, and we did not want to introduce breaking changes in this PR.
I do agree that this PR should be unified to a single `onDependency` method.